app/retail/emails.py

```diff
@@ -993,16 +994,24 @@ def render_new_bounty_roundup(to_email):
         'link_copy': 'View more',
     }, ]
 
-    sponsor = {
-        'name': 'Blockchain Job Kit',
-        'title': 'Blockchain Developer Job Kit',
-        'image_url': 'https://s3.us-west-2.amazonaws.com/gitcoin-static/jDSk7ZTfpY19PWdwwsk8puNd.png',
-        'link': 'http://bit.ly/EthDevKit',
-        'cta': 'View now',
-        'body': [
-            'Blockchain Developer Job Kit - See who’s hiring, salary info, and dev skills required'
-        ]
-    }
+    sponsor_response = (requests.get(settings.CODEFUND_NEWSLETTER_URL)
```

Comment:
note that this will run 25k times per mgmt command (and will grow bounded to the size of the email list).  it might be better performance and reliability-wise to call it once, and then to store the results in memory per-management command execution...

Reply:
Yeah good point - we should probably render the entire template once for better performance since theres no user-specific info in it.